toolchain/check/handle_operator.cpp

```diff
@@ -4,20 +4,46 @@
 
 #include "toolchain/check/context.h"
 #include "toolchain/check/convert.h"
+#include "toolchain/check/operator.h"
 #include "toolchain/check/pointer_dereference.h"
 #include "toolchain/diagnostics/diagnostic_emitter.h"
 
 namespace Carbon::Check {
 
+auto HandleUnaryOperator(Context& context, Parse::NodeId node_id, Operator op)
```

Comment:
```suggestion
// Common logic for unary operator handlers.
static auto HandleUnaryOperator(Context& context, Parse::NodeId node_id, Operator op)
```

The `static` is the more important part of my suggestion.